tsconfig.types.json

```diff
@@ -13,6 +13,11 @@
     "src/core/public/index.ts",
     "src/plugins/data/server/index.ts",
     "src/plugins/data/public/index.ts",
+    "src/plugins/embeddable/server/index.ts",
+    "src/plugins/embeddable/public/index.ts",
+    "src/plugins/expressions/server/index.ts",
```

Comment:
non uiActions changes expected?

Reply:
yeah, the expressions are merged but i didn't notice this file before. everything was still generated correctly but i assume because of dependencies between data and expressions, so this should actually be there.